apps/web/src/components/molecules/confirmCard/confirmCard.module.scss

```diff
@@ -0,0 +1,94 @@
+@use "@/colors.module.scss" as colors;
+
+.root {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 10000;
+  background-color: rgba($color: #000000, $alpha: 0.5);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  @media (max-width: 600px) {
+    align-items: flex-end;
+  }
+
+  .content {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
wydaje mi się że bez tego też to będzie wycentrowane, kontener ma justify-content i align-items na center, te zaznaczone style można usunać